synapse/replication/tcp/streams/_base.py

```diff
@@ -88,8 +88,7 @@
     "TagAccountDataStreamRow", ("user_id", "room_id", "data")  # str  # str  # dict
 )
 AccountDataStreamRow = namedtuple(
-    "AccountDataStream",
-    ("user_id", "room_id", "data_type", "data"),  # str  # str  # str  # dict
+    "AccountDataStream", ("user_id", "room_id", "data_type")  # str  # str  # str
```

Comment:
I assume you checked that nothing actually uses the `data` field in `synapse.replication.slave.storage`?

Reply:
Mmm, it does seem to be used here:

https://github.com/matrix-org/synapse/blob/c66a06ac6b69b0a03f5c6284ded980399e9df94e/synapse/replication/slave/storage/events.py#L108

Perhaps I could just wrap that in a `if "data" in row: return`?

Reply:
That just looks to be for the "events" stream, rather than the account data?

Reply:
Oh, true. I think I thought Event was generic enough to cover anything. But it looks like it is just a separate stream. Think we're good then?

Reply:
I think we are!